src/reducer/ThemeReducer.js

```diff
@@ -0,0 +1,12 @@
+export const ThemeReducer = (prevTheme, { type, payload }) => {
```

Comment:
Shall we use default export here?
Since there will be only one ThemeReducer

Reply:
Not much of a problem though, ok I will change it to default export